csr/csr.go

```diff
@@ -50,6 +50,12 @@ func VerifyCSR(csr *x509.CertificateRequest, maxNames int, keyPolicy *goodkey.Ke
 	if err := csr.CheckSignature(); err != nil {
 		return errors.New("invalid signature on CSR")
 	}
+	if len(csr.EmailAddresses) > 0 {
+		return errors.New("CSR contains one or more emailAddress fields")
+	}
+	if len(csr.IPAddresses) > 0 {
+		return errors.New("CSR contains one or more iPAddress fields")
```

Comment:
On thinking about this some more, I realized that it makes more sense to use human-friendly language than to do this confusing thing that references a spec most of our users have never read. Updated.